traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -2202,3 +2202,18 @@ func ServiceCategoryExists(tx *sql.Tx, name string) (bool, error) {
 	}
 	return true, nil
 }
+
+// GetASNInfo confirms whether the asn exists, and an error (if one occurs).
+func GetASNInfo(tx *sql.Tx, id string) (bool, error) {
```

Comment:
As this function does not get the ASNInfo and it only confirms the existence of the ASN, should we rename the funcion to something like `ASNExists`

Reply:
Agreed.